web/angular-wallet/angular.json

```diff
@@ -97,8 +100,11 @@
                     }
                 },
                 "serve": {
-                    "builder": "@angular-devkit/build-angular:dev-server",
+                    "builder": "@angular-builders/custom-webpack:dev-server",
                     "options": {
+                        "customWebpackConfig": {
+                            "path": "./extra-webpack.config.js"
+                        },
```

Comment:
sadly we have to use this custom webpack config because the `namicorn` lib uses `stream`, `os`, `crypto`, etc and the default angular builder does not support these node modules.